src/reducers/counter.js

```diff
@@ -3,29 +3,52 @@
 import { Actions } from "../actions/counter";
 
 const counter = {
```

Comment:
Is this state now unused? I couldn't find it used in the code, if it's unused then it should be removed.